index.html

```diff
@@ -6,6 +6,6 @@
     <title>Nep huiswerkopdracht</title>
 </head>
 <body>
-
+<h1>making changes</h1>
```

Comment:
Deze liever in <h2>?

Reply:
h2 (html code laat hij dus niet zien?)

Reply:
Nee inderdaad, er staat nu een vraagteken logo